docs/team/zhaojj2209.md

```diff
@@ -5,27 +5,41 @@ title: Zhao Jingjing's Project Portfolio Page
 
 ## Project: Fine$$e
 
+### Overview
+
 Fine$$e is a desktop finance tracker that allows tertiary students to better manage their finances, cultivating good financial habits such as saving. It is optimised for CLI users so that expenses and income can be tracked efficiently by typing in commands. It is written in Java, and has a GUI created with JavaFX.
 
-Given below are my contributions to the project.
+### Summary of Contributions
+
+* **Major Enhancement**: Added the monthly budget feature to the finance tracker.
+  * What it does: With the addition of the monthly budget, the finance tracker allows the user to set their desired monthly expense limit and monthly savings goal, and calculates the user's remaining budget and current savings.
+  * Justification: This feature significantly enhances the capabilities of the finance tracker by allowing the user to track their progress in their savings, and paves the way for the application's analytics feature.
+  * Highlights: This feature required heavy use of the observer design pattern as all values in the monthly budget have to be observed by the UI so that the values and calculations can be updated immediately upon any changes.
 
-* **New Feature**:
-  * What it does:
-  * Justification:
-  * Highlights:
-  * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+* **Major Enhancement**: Revamped the finance tracker's `find` command.
```

Comment:
Should `Enhancement` be lowercase to remain consistent with the rest of the bullet points?